spec-main/api-web-frame-spec.ts

```diff
@@ -37,4 +38,18 @@ describe('webFrame module', () => {
     expect(words.sort()).to.deep.equal(['spleling', 'test', 'you\'re', 'you', 're'].sort());
     expect(callbackDefined).to.be.true();
   });
+
+  it('enable text zoom in webview', async () => {
```

Comment:
We also need to test that a webview can call these new methods - see `spec/fixtures/pages/webview-custom-zoom-level.html`